src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -313,13 +313,19 @@ internal void Update(double number)
 
                 case AggregationType.Histogram:
                     {
-                        int i;
-                        for (i = 0; i < this.histogramBuckets.ExplicitBounds.Length; i++)
+                        // If -inf is the first bound, +inf will be the second of two bounds
+                        // in which case there is only a single bucket, so set the bucket
+                        // index to increment should be zero.
+                        int i = 0;
+                        if (!double.IsNegativeInfinity(this.histogramBuckets.ExplicitBounds[0]))
```

Comment:
Instead of checking for `IsNegativeInfinity`, just check if `this.histogramBuckets.RunningBucketCounts == 1`.

Reply:
Ya, that's a more sensible check to make